cmd/cli/main.go

```diff
@@ -104,16 +104,25 @@ func main() {
 	rootCmd.AddCommand(cliSVC.NewAlgorithmCmd())
 	rootCmd.AddCommand(cliSVC.NewDatasetsCmd())
 	rootCmd.AddCommand(cliSVC.NewResultsCmd())
-	attestaionCmd := cliSVC.NewAttestationCmd()
-	rootCmd.AddCommand(attestaionCmd)
+	attestationCmd := cliSVC.NewAttestationCmd()
+	rootCmd.AddCommand(attestationCmd)
 	rootCmd.AddCommand(cliSVC.NewFileHashCmd())
 	rootCmd.AddCommand(cliSVC.NewAddMeasurementCmd())
 	rootCmd.AddCommand(cliSVC.NewKeysCmd())
 	rootCmd.AddCommand(cliSVC.NewCABundleCmd(directoryCachePath))
 
 	// Attestation commands
-	attestaionCmd.AddCommand(cliSVC.NewGetAttestationCmd())
-	attestaionCmd.AddCommand(cliSVC.NewValidateAttestationValidationCmd())
+	attestationCmd.AddCommand(cliSVC.NewGetAttestationCmd())
+	attestationCmd.AddCommand(cliSVC.NewValidateAttestationValidationCmd())
+
+	// Flags
+	rootCmd.PersistentFlags().StringVarP(
```

Comment:
this flag should not be in the root command but rather in the key generation command